ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/less/button.less

```diff
@@ -76,6 +76,22 @@
 		background: #efaa9b;
 	}
 }
+.btn-phone-call  {
```

Comment:
@AyushHCL : pl make the changes to UI for call variation as discussed . 

Expected UI : 
![image](https://user-images.githubusercontent.com/38690824/82777422-ba268f80-9e6b-11ea-951c-62da1cd651a5.png)

Current UI : 
![image](https://user-images.githubusercontent.com/38690824/82777468-d7f3f480-9e6b-11ea-82b5-6ecbd49cdc7d.png)